api/src/main/java/org/pmiops/workbench/reporting/ReportingService.java

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.reporting;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.pmiops.workbench.model.ReportingSnapshot;
+
+public interface ReportingService {
+  @VisibleForTesting
+  ReportingSnapshot getSnapshot();
```

Comment:
these are both `public` in the impl

Reply:
right, It's only in the interface (thus public when overridden) for testing. But this is already changed in the next batch of code.